Libraries/LibJS/Runtime/Object.cpp

```diff
@@ -123,7 +123,7 @@ Value Object::get_own_property(const Object& this_object, PropertyName property_
     return value_here;
 }
 
-Value Object::get_own_properties(const Object& this_object, GetOwnPropertyMode kind, u8 attributes) const
+Value Object::get_own_properties(const Object& this_object, GetOwnPropertyMode kind, PropertyAttributes attributes) const
```

Comment:
Now that we're looking at this, this API seems a bit goofy.

What does it mean to `get_own_properties` with a set of `attributes` anyway?

This is really just a flag that filters based on the `enumerable` attribute AFAIK so maybe it can be a `bool` instead?